routes/account/projectOwner.js

```diff
@@ -29,8 +29,8 @@ const // /api/account/project-owner
           if (lastName) {
             userUpdate.lastName = lastName;
           }
-          if (image_url) {
-            userUpdate.image_url = image_url;
+          if (profile_picture_url) {
+            userUpdate.profile_picture_url = profile_picture_url;
```

Comment:
I believe on the left side it should stay as userUpdate.image_url - cuz this is the object that will be saved in db